src/v3/src/components/WebAuthNAutofill/WebAuthNAutofill.test.tsx

```diff
@@ -79,7 +79,7 @@ describe('WebAuthNAutofill', () => {
     });
   });
 
-  it.each([ABORT_REASON_CLEANUP, ABORT_REASON_WEBAUTHN_AUTOFILLUI_STEP_NOT_FOUND])('should not call the setMessage method for ignored error "%s"', async (err) => {
```

Comment:
Add a comment here inline (might be worth reformatting the array into separate lines) to help a reader understand why `{}` is here so they don't inadvertently try to remove it.

Reply:
Updated, thanks for the suggestion.